food-truck-api/src/main/java/food/truck/api/routes/RouteService.java

```diff
@@ -143,4 +152,16 @@ public boolean userOwnsLocation(User u, long locationId) {
         var loc = routeLocationRepository.findById(locationId);
         return loc.isPresent() && loc.get().getRoute().getTruck().getUserId().equals(u.getId());
     }
+
+    /**
+     * Translates an instant into its specified time on the day of the epoch. Allows
+     * comparison of instants.
```

Comment:
Instant.isBefore(x) would be easier and more reliable

Reply:
Whoops, that was an artifact from trying to fix what turned out to be a DB config issue, and I just forgot about it. I'll dispose of it.